### PR TITLE
update license

### PR DESCRIPTION
**Pull request by qiaojialin**
Jira issue:

https://issues.apache.org/jira/projects/IOTDB/issues/IOTDB-45?filter=allopenissues

Replace org.json with alibaba.fastjson
Remove netty dependency
Update license and notice file
Add license in Licenses/

### CODE REVIEW

**Comment by MyXOF** [comment]
You should send an email to our community to check whether you can add hive apache license to our project

File: example/rocketmq/pom.xml [hunk 1]
```diff
@@ -34,16 +34,7 @@
...
-                <exclusion>
-                    <groupId>io.netty</groupId>
-                    <artifactId>netty-common</artifactId>
-                </exclusion>
-                <exclusion>
-                    <groupId>io.netty</groupId>
-                    <artifactId>netty-buffer</artifactId>
-                </exclusion>
-            </exclusions>
+            <version>0.8.0-SNAPSHOT</version>
```
[the remark this answers is not in the document]
Comment (MyXOF) [reply example/rocketmq/pom.xml]: use `<version>${project.version}</version>` ?

**Comment by jixuan1989** [comment]
> LGPL 2.1 seems not be used.

And pls make sure that EDL license file is included.

File: example/rocketmq/pom.xml (hunk 1 above)
[the remark this answers is not in the document]
Comment (jixuan1989) [reply example/rocketmq/pom.xml]: No, you can omit the declaration in this case..

File: tsfile/src/test/java/org/apache/iotdb/tsfile/write/WriteTest.java [hunk 2]
```diff
@@ -98,10 +97,12 @@ public void prepare() throws IOException, WriteProcessException {
     if (errorFile.exists()) {
       errorFile.delete();
     }
-    JSONObject emptySchema = new JSONObject("{\"delta_type\": \"test_type\",\"properties\": {\n"
+
+    JSONObject emptySchema = JSON.parseObject("{\"delta_type\": \"test_type\",\"properties\": {\n"
         + "\"key1\": \"value1\",\n" + "\"key2\": \"value2\"\n" + "},\"schema\": [],}");
-    measurementArray = new JSONObject(new JSONTokener(new FileReader(new File(schemaFile))))
-        .getJSONArray(JsonFormatConstant.JSON_SCHEMA);
+    InputStream inputStream = new FileInputStream(schemaFile);
```
Comment (jt2594838) [inline tsfile/src/test/java/org/apache/iotdb/tsfile/write/WriteTest.java]: Is the stream closed?

File: example/rocketmq/pom.xml (hunk 1 above)
[the remark this answers is not in the document]
Comment (qiaojialin) [reply example/rocketmq/pom.xml]: ok

**Comment by qiaojialin** [comment]
> LGPL 2.1 seems not be used.

removed

**Comment by qiaojialin** [comment]
> > LGPL 2.1 seems not be used.
> 
> And pls make sure that EDL license file is included.

@jixuan1989 added

**Comment by qiaojialin** [comment]
> Several InputStreams in tests seem left unclosed.

I have put inputstream into a try()

**Pull request merged**